src/components/Plants/Plant.tsx

```diff
@@ -1,3 +1,6 @@
+import usePlantPics from './usePlantPics';
+import { getPhotosUrls } from '../../utils/flickr';
```

Comment:
I tried to setup absolute paths, but it’s not as “super simple” as they claim: https://nextjs.org/blog/next-9-4#absolute-imports-and-aliases

Reply:
can we take a look at it?